src/Resources/config/services.php

```diff
@@ -2,17 +2,11 @@
 
 declare(strict_types=1);
 
-/*
- * The MIT License (MIT)
- *
- * Copyright (c) 2014-2020 Spomky-Labs
- *
- * This software may be modified and distributed under the terms
- * of the MIT license.  See the LICENSE file for details.
- */
-
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 use Yoanbernabeu\AirtableClientBundle\AirtableClient;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use function Symfony\Component\DependencyInjection\Loader\Configurator\service;
 
 return static function (ContainerConfigurator $container): void {
     $container = $container->services()->defaults()
```

Comment:
:bulb: Not related to you PR but if we want to be "clean" we should work with interface, create an AirtableClientInterface which reference to this service https://symfony.com/doc/current/service_container/autowiring.html#working-with-interfaces

Reply:
But as the service is public it also can be done by the end user so it's ok to keep it like this 👍🏽